documentation/master.adoc

```diff
@@ -6,8 +6,12 @@ include::modules/address-model.adoc[leveloffset=+1]
 
 include::modules/address-space.adoc[leveloffset=+1]
 
+include::modules/installing-openshift.adoc[leveloffset=+1]
```

Comment:
My initial instinct is that the we need more than one 'book' here. For the console help, I don't think it makes sense to include the installation guide. Also in terms of sequence, I think for getting started you would want to start with the installation.

Reply:
@grs, yes, we should add additional master.adoc files for each guide.

Reply:
To update my comment above: please see https://github.com/EnMasseProject/enmasse/pull/364
I think that the content below belongs in the new service_admin/master.adoc file?